pyvrp/Population.py

```diff
@@ -118,13 +167,17 @@ def remove(self, individual: Individual):
         ValueError
             When the given individual could not be found in the subpopulation.
         """
-        for _, _, prox in self:  # remove individual from other proximities
+        for (
+            _,
+            _,
+            prox,
+        ) in self._items:  # remove individual from other proximities
```

Comment:
Can we keep something like the original here? The one-line for loop was easier to read than this five-line thing, IMHO.